R/select.R

```diff
@@ -1,60 +1,174 @@
-#' @noRd
-#' Internal functions to select from QTables using Combo Box
-#' R Controls, which are reprene
-#' @return the number of tables supplied by the user using Table - Select GUI controls
-nTables <- function()
-    length(ls(pattern = "^formTables[0-9]+$", envir = .GlobalEnv)) - 1
-
-# Subsets a table based on values of combo.boxes
-selectFromTableUsingComboBoxes <- function(table, table.orig, combo.boxes, dim = 1)
```

Comment:
This will break existing outputs for users when it is removed?

Reply:
We haven't released a version of Select that uses `verbs` yet, that function only appears in the Standar R code, so there should be no issues.